src/Plugin.php

```diff
@@ -182,9 +185,26 @@ public function onDependenciesChangedEvent()
                 $io->write(sprintf('<info>%s</info>', self::MESSAGE_NOTHING_TO_INSTALL));
             }
         } else {
-            $exitCode = 1;
-            if ($isVerbose) {
-                $io->write(sprintf('<info>%s</info>', self::MESSAGE_NOT_INSTALLED));
+            $isDev = array_key_exists(self::PLUGIN_NAME, $this->composer->getPackage()->getDevRequires());
```

Comment:
:point_up: This looks like it only takes the `dev` requires into account, but the issue isn't limited to `dev` requires.

Reply:
Fixed.